ui/apps/platform/src/Components/PatternFly/BacklogListSelector.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { Key, ReactNode } from 'react';
+import {
+    Badge,
+    Button,
+    EmptyState,
+    EmptyStateIcon,
+    EmptyStateVariant,
+    Flex,
+    FormGroup,
+    Title,
+} from '@patternfly/react-core';
+import { CubesIcon, MinusCircleIcon, PlusCircleIcon } from '@patternfly/react-icons';
+
+import {
+    BaseCellProps,
+    TableComposable,
+    TableVariant,
+    Tbody,
+    Td,
+    Tr,
+} from '@patternfly/react-table';
+
+type BacklogTableProps<Item> = {
+    type: 'selected' | 'deselected';
+    label: string | undefined;
+    items: Item[];
+    listAction: (item: Item) => void;
+    rowKey: (item: Item) => Key;
+    cells: {
+        name: string;
+        render: (item: Item) => ReactNode;
+        width?: BaseCellProps['width'];
+    }[];
+    buttonText: string;
+};
+
+function BacklogTable<Item>({
+    type,
+    label,
+    items,
+    listAction,
+    rowKey,
+    cells,
+    buttonText,
+}: BacklogTableProps<Item>) {
+    const actionIcon =
+        type === 'selected' ? (
+            <MinusCircleIcon color="var(--pf-global--danger-color--200)" />
+        ) : (
+            <PlusCircleIcon color="var(--pf-global--primary-color--100)" />
+        );
+    return (
+        <FormGroup
+            label={
+                <>
+                    {label}
+                    <Badge className="pf-u-ml-sm" isRead>
+                        {items.length}
+                    </Badge>
+                </>
+            }
+        >
+            {items.length > 0 ? (
+                <TableComposable aria-label={label} variant={TableVariant.compact}>
```

Comment:
Can you inspect table cells to see why centered alignment in responsive view?

Here is an example from /main/access-control/permission-sets for comparison:
![Permission_sets](https://user-images.githubusercontent.com/11862657/198709255-3d170292-ce2e-4ec7-9e98-8c19d9903079.png)